src/styles/_components/_inputs.scss

```diff
@@ -544,16 +544,20 @@ $checkbox-size: 32px;
 @mixin select() {
     height: $input-height;
     @include height-modifiers();
-    border: 0;
     font-size: em(18px);
     color: color('text', 'input');
-    padding: 0 14px;
-    padding-right: 46px;
+    padding: 0 46px 0 14px;
+    border: 2px solid color('borders');
+    border-radius: 4px;
+
     will-change: border;
     -webkit-appearance: none;
-    border: 2px solid color('borders');
-    border-top-right-radius: 4px;
-    border-bottom-right-radius: 4px;
+    -moz-appearance: none;
```

Comment:
please change to:

`appearance: none;`

this way auto-prefixer will kick in and generate all necessary versions.